src/components/NavBar.js

```diff
@@ -2,15 +2,22 @@ import { Link } from 'react-router-dom';
 
 function NavBar() {
   return (
-    <nav className="Nav-bar flex blue">
-      <div className="Links flex">
-        <h1>Bookstore CMS</h1>
-        <ul className="flex">
-          <li><Link to="/">BOOKS</Link></li>
-          <li><Link to="/categories">CATEGORIES</Link></li>
-        </ul>
-      </div>
-      <div className="profile flex blue">Profile</div>
+    <nav className="nav-bar">
+      <h1>Bookstore CMS</h1>
+      <ul className="nav-links">
+        <li className="link">
+          {' '}
+          <Link className="Books" to="/">
+            BOOKS
+          </Link>
+        </li>
+        <li className="linked">
+          {' '}
+          <Link className="category" to="/Categories">
+            CATEGORIES
+          </Link>
+        </li>
+      </ul>
```

Comment:
- [ ] As requested by the previous reviewer please make sure that you design matches the original design. I attach the design below please make sure to implement it.
### Your design
![his](https://github.com/Nuga0909/bookstore3/assets/81727988/9c7d03a8-4cc7-4e1e-9451-7dd6f345c6a4)

### Original design
![figma](https://github.com/Nuga0909/bookstore3/assets/81727988/e193da22-1818-4a0a-a0d2-5781c56dc99b)